client/rhel/spacewalk-client-tools/src/bin/spacewalk-update-status.py

```diff
@@ -45,7 +45,7 @@ def __build_status_report():
                 pass
 
         # We need to fit into xmlrpc's integer limits
-        if status_report['uptime'][1] > long(2)**31-1:
+        if status_report['uptime'][1] > int(2)**31-1:
```

Comment:
Will this still work with python2? If yes, fine.

Reply:
Tested (see comment below from @mantel itself) :-)